xml/signals/basic-enhanced/index.shtml

```diff
@@ -0,0 +1,86 @@
+      </p>
+
+      <p><u><b>USING SPEED TO SELECT SECONDARY DIVERGING ASPECTS:</u></b> When using the three-head mast contained in this Basic Enhanced signal system, set the speed of the diverging route to be signaled by the bottom head to "Medium".  This is easily accomplished by creating a dummy block whose speed is set to "Medium" and including it in that route's logic with a test setting of "Any State".  The dummy block is not part of the actual layout and does not need to be placed on the layout panel.
+	</p>
+
+	<p>In JMRI, a route is either Normal or Diverging.  If a route contains even one turnout in Thrown state, then that entire route is Diverging.  If a route contains two turnouts or two crossovers, that means there are two diverging paths.  JMRI uses a route's speed to determine which head on a mast is used to signal the second diverging path.  In the case of the three head mast contained in the Basic Enhanced signal system, the bottom head has its speed set to Medium (slower than Limited) and is intended to signal this two-turnout case.  It can be used to indicate the route will take the second of two successive, closely spaced turnouts, or that the path passes through both closely spaced crossovers, as in the case of a  three-track crossover move.  Usage of the dummy block set to Medium speed as discussed above is an easy method to inform JMRI to use the bottom head to signal this secondary route.
+
+      </p>
+
+      <p><u><b>DISCUSSION ON ROUTE SPEED:</u></b>  The speed of a given route between two Signal Masts is equal to the slowest speed assigned to any element in the route.  When a block is created, JMRI assigns the Global Normal speed to it.  By default this is "Normal" speed.  When a turnout is created, JMRI assigns the Global Normal speed to its Closed path and the Global Thrown speed to its Thrown path.  These by default are "Normal" and "Restricting", respectively.  The usage recommendation for the Basic Enhanced signal system is the set the Global Thrown speed to "Limited."  See above for instructions.  If a route between two masts contains blocks and turnouts that are all Closed, then the speed of that route is the Global Normals peed.  If a route contains a turnout in the Thrown state, the speed of that route is the Global Thrown speed.  A route's speed can be modified individually by overriding the defaults.  JMRI provides several options for doing this. For example, the speeds for a turnout's paths or a block can be individually set, or a virtual element can be created and included in that path to set the path's speed.  Remember that the slowest speed of any element in the route sets the speed of the entire route.  The speed names (Restricted, Medium, etc) and their relative magnitude are found in file xml/signals/signalSpeeds.xml.
```

Comment:
peed should be speed.

HTML tags are nested and need to be reversed when closed.  `<u><b>` is closed by `</b></u>`.

Long lines should be wrapped at about column 100.

The indents are a mixture of tabs and spaces.

Reply:
Thanks for doing this!  You might already know this, but as it's your first PR I think I should mention that you can make the changes that Dave Sand suggested, and then just commit and push them.  They'll automatically get added to the PR.  Thanks again!

Reply:
Bob:  Thank you.  No, I didn't know about the commit being automatically added to the PR.  I recommitted the file.  I'm using Github on the web rather than desktop, so hopefully there is no push required.

Dave:  Thank you for your review and corrections list.  I made the corrections for HTML tag order, line length and spelling.  This version of index.shtml was edited with VSCode with "render whitespace" on.  Assuming I'm reading the display correctly, all indents are now solely via spaces.

Reply:
I have not read the text closely yet, but there are still a couple of warnings when using `tidy`.
```
line 24 column 17 - Warning: missing <li>
line 27 column 27 - Warning: missing <li>
```

These are caused by the embedded `<ul>` content.

![Screenshot 2024-11-11 at 9 43 55 PM](https://github.com/user-attachments/assets/5a70f4d7-ddfd-46c7-b30d-469ca51ffadd)

This content goes within the scope of the `<li>` so the closing `</li>` goes after the closing `</ul>`.  The closing </li> for line 23 goes after line 35.  The one at 26 goes after line 32.

You can see that there are still some tabs.  VSCode should have an option to display special characters.

Reply:
Hi Dave:  I believe I have resolved all the tab vs space indents.  Yes VSCode allows for displaying tab characters.  It was just with the small print on the dark background, the tiny right-arrow character was difficult to distinguish from the dots representing spaces.  Lighter background and a few zooms later problem resolved.  I used the regex search to locate the tab character to make sure and found another instance in another file, which is also fixed.  The embedded list issues have also been fixed.  When I resolve another question on <imagelink> keyword I'll upload latest versions.  Thanks for your patience and help.  --John.